tests/scripts/local_monitoring_tests/rest_apis/test_list_datasources.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import json
+import sys
+
+sys.path.append("../../")
+
+from helpers.fixtures import *
+from helpers.kruize import *
+from helpers.utils import *
+from helpers.list_datasources_json_validate import *
+
+
+@pytest.mark.sanity
+def test_list_datasources_without_parameters(cluster_type):
+    """
+    Test Description: This test validates datasources API without parameters
+    """
+    form_kruize_url(cluster_type)
+
+    # Get the datasources name
+    datasource_name = None
+    response = list_datasources(datasource_name)
+
+    list_datasources_json = response.json()
+
+    assert response.status_code == SUCCESS_200_STATUS_CODE
```

Comment:
Please include validation of the response message as well

Reply:
In this case accessing `list_datasources_json['message']` is resulting in  `KeyError: 'message'` as the expected response is a JSON - with list of datasources, have added validation of the output json.